src/components/MapView/index.tsx

```diff
@@ -311,13 +314,23 @@ function MapView({ classes }: MapViewProps) {
 
   useEffect(() => {
     dispatch(loadAvailableDates());
+
     const displayBoundaryLayers = getDisplayBoundaryLayers();
+    /*
+      reverse the order off adding layers so that the first boundary layer will be placed at the very bottom,
+      to prevent other boundary layers being covered by any layers
+    */
+    // eslint-disable-next-line fp/no-mutating-methods
+    const reversedBoundaryLayer = displayBoundaryLayers
```

Comment:
you can probably do it a bit simplified like this:

const reversedBoundaryLayer = displayBoundaryLayers
      .reverse()
      .map(layer => displayBoundaryLayers.filter(boundary => boundary.id === layer.id)[0]);

Reply:
Yes it should work with just reverse(). I thought it didn't work before because it didn't change when I was debugging. Just realized that I have to clone the variable first before debugging because it destruct the initial variable 🙈